RestingStateStats/Compiled_RestingStateStats/readme.txt

```diff
@@ -2,27 +2,27 @@ MATLAB Compiler
 
 1. Prerequisites for Deployment 
 
-. Verify the MATLAB Compiler Runtime (MCR) is installed and ensure you    
-  have installed version 8.1 (R2013a).   
+. Verify the MATLAB Runtime is installed and ensure you    
+  have installed version 9.1 (R2016b).   
 
-. If the MCR is not installed, do the following:
+. If the MATLAB Runtime is not installed, do the following:
```

Comment:
This should probably say "If you have this version of MATLAB installed, but the MATLAB Runtime is not installed".

Reply:
This `readme.txt` file is generated by the MATLAB compiler. So we don't have control over it to change it's contents.